cmd/chisel/cmd_cut.go

```diff
@@ -91,11 +91,20 @@ func (cmd *cmdCut) Execute(args []string) error {
 			Suites:     archiveInfo.Suites,
 			Components: archiveInfo.Components,
 			CacheDir:   cache.DefaultDir("chisel"),
+			Priority:   archiveInfo.Priority,
+			Pro:        archiveInfo.Pro,
 		})
 		if err != nil {
-			return err
+			if err != archive.ErrNoArchiveCredentials {
+				return err
+			}
+		} else {
+			archives[archiveName] = openArchive
 		}
-		archives[archiveName] = openArchive
+	}
+
```

Comment:
As the code block above populates `archives` and the block here checks it, I think they're strongly-linked enough to merit removing this blank line